src/pybind/mgr/ssh/Vagrantfile

```diff
@@ -0,0 +1,63 @@
+Vagrant.configure("2") do |config|
```

Comment:
Hm. Adding a Vagrantfile to set up a development environment in the Ceph tree? I don't know, if this is a good idea. 

I personally would prefer to have this located our of the main Ceph repository.

Reply:
Playing the devil's advocate here: but then `vstart.sh` should be located elsewhere, too ;)

I personally am actually in favor of having development-related scripts and tools included in the main git repo, as it ensures that they are in sync with changes to the codebase and developers are encouraged to use and improve them. However, I'd suggest a more prominent place for them, e.g. in a toplevel directory `vagrant` or so.

Reply:
yeah i'm happy to have any developer scripts in tree!

Reply:
Hm. when looking at more sophisticated alternatives used by the [dashboard](https://github.com/ricardoasmarques/ceph-dev-docker/) and [DeepSea](https://github.com/openSUSE/vagrant-ceph), they don't really fit into a standard mgr module.

Especially the dashboard environment has a longer history and went from vagrant based setups to a docker based one. 

If the SSH orchestrator is going to get popular, I expect the development environment to grow significantly. 

On the other hand, this is a nit and not going to block this PR.

Reply:
@sebastian-philipp i'm def. open to solutions for managing more complex dev setups. vagrant was useful since it does ssh and networking setup automatically, and ssh in docker is an anti-pattern. i think we can sit on this until we figure out how testing is gonna work. it may end up being useful for that.